CMakeLists.txt

```diff
@@ -14,6 +14,10 @@ set_target_properties(ArborX PROPERTIES INTERFACE_COMPILE_FEATURES cxx_std_14)
 # within a subdirectory.
 add_dependencies(ArborX record_hash)
 
+# Refer to the alias target in examples and benchmarks so they can be built
+# against an installed ArborX
+add_library(ArborX::ArborX ALIAS ArborX)
```

Comment:
I'm a bit confused now. Prior to this patch, external applications already used `ArborX::ArborX`. Where was this set? Was it done automatically by CMake? If yes, does it mean that it is now possible to link against `ArborX::ArborX::ArborX`?

Reply:
This one is for the library built, the previous one was for the installed library AFAICT.

Reply:
Ah, good point.